client/homebrew/pages/errorPage/errorPage.jsx

```diff
@@ -5,9 +5,11 @@ const Markdown   = require('../../../../shared/naturalcrit/markdown.js');
 const ErrorIndex = require('./errors/errorIndex.js');
 
 const ErrorPage = ({ brew })=>{
+	
 	// Retrieving the error text based on the brew's error code from ErrorIndex
 	const errorText = ErrorIndex({ brew })[brew.HBErrorCode.toString()] || '';
-
+	console.log(errorText);
```

Comment:
Not sure we need to reprint the same error in the console if it's already being displayed to the user in the page.